incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -95,9 +99,11 @@ spec:
         - containerPort: 14269
           name: admin
           protocol: TCP
+        {{- if hasKey .Values.collector.service "zipkinPort" }}
```

Comment:
We should not use these `if` statements here since in the default values there is `zipkinPort` set. This means that all of the `if` statements will always be set to true. Please, remove the `if` statements.